packages/form/components/controls/RadioGroup.astro

```diff
@@ -29,6 +30,7 @@ const options = control.options.map((option: string | ControlOption) => {
 				name={control.name}
 				value={option.value}
 				checked={option.value === control.value}
+				disabled={readOnly}
```

Comment:
Can you also add a property `readonly`, same scenario as the checkbox?

This is so we later can style disabled and readonly differently. :)

Reply:
The radiobutton and the checkbox can still be toggled if the readonly flag is passed, they only are locked with the disabled option, it is ok to they still be changed even if they are readonly? Or we can also try another method to keep it readonly, what do you think?

Reply:
I'm thinking let's have both attributes disabled and readonly

```typescript
disabled={readOnly || null}
readonly={readOnly || null}
```

Also I think having `|| null` like above is needed so the property will not appear when for example we get undefined

Reply:
Ah probably the select element doesn't have it. They do have differences. That's okay.